src/components/HomeScreenHeader.tsx

```diff
@@ -1,11 +1,12 @@
 import { StackNavigationProp } from '@react-navigation/stack'
```

Comment:
Because I requested it as the previous name `HeaderWithMenu` was definitely misleading. This is definitely home screen specific as the option `Manage disciplines` does not make sense somewhere else.